### PR TITLE
Regroup list of public repos

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* add `adyen-networking-ios`
* add `adyen-3ds2-ios`
* add `adyen-3ds2-android`
* rename `Adyen React Native [Beta]` -> `Adyen React Native [BETA]`
* regroup everything
* remove `Adyen` everywhere
* convert all labels to **sentence case**
* mark "API library for Apex" and "PHP webhook helper module for payment integrations" as **BETA**
* mark "Plugin for feast to run an offline store on Spark" as **Discontinued**

### CODE REVIEW

File: profile/README.md
Comment (candemiralp) [inline profile/README.md]: Can we move Magento 2 on top of the Shopware 6?

File: profile/README.md
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Payment Plugins
```

File: profile/README.md
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Integration Plugins
```

File: profile/README.md
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Other Repos
```

File: profile/README.md
Comment (candemiralp) [inline profile/README.md]:
```suggestion
### Other Repos
```

**Review by michaelpaul — APPROVED** [review]

File: profile/README.md
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: fixed in [bbac6d7](https://github.com/Adyen/.github/pull/7/commits/bbac6d73fc6f0a41baf16258b2a3ac188c4a4185)

File: profile/README.md
Comment (AdrienneBrieffies) [reply profile/README.md]: At Adyen we use sentence case for headings and titles ("Payment plugins"), not title case ("Payment Plugins").

File: profile/README.md
Comment (AdrienneBrieffies) [reply profile/README.md]: Please use sentence case ("Other repos") and consider using the full word: Other repositories

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Duplicate

**Review by wboereboom — APPROVED** [review]

File: profile/README.md
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Other repos
```

File: profile/README.md
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Payment plugins
```

File: profile/README.md
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
#### API libraries
```

File: profile/README.md
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
#### Drop-in and components
```

File: profile/README.md
Comment (rikterbeek) [inline profile/README.md]:
```suggestion
### Integration plugins
```

Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed as 'Other repos' on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

File: profile/README.md
Comment (descorp) [reply profile/README.md]: Fixed on [cab4990](https://github.com/Adyen/.github/pull/7/commits/cab4990013fbcba82db1356b6602f758dec59390)

**Review by michaelpaul — APPROVED** [review]

**Comment by rikterbeek** [comment]
HI @descorp,

Are you done with the changes if so can you remove the [WIP].

Regards,
Rik

**Comment by descorp** [comment]
> HI @descorp,
> 
> Are you done with the changes if so can you remove the [WIP].
> 
> Regards, Rik

Waiting for review from 

| Team | Approve |
| --- | --- |
| (Tech Writers) | ✅ |
| @Adyen/integration-tools | ✅ | 
| @Adyen/plugins-partners | ✅ | 
| @Adyen/ipp-mobile-ios | ✅ |

**Review by kaphacius — APPROVED** [review]

**Review by Morerice — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

**Review by peterojo — APPROVED** [review]

**Pull request merged**
